design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,147 @@
+type Mutation {
+    requestNegotiableQuote(
+        cart_id: ID! @doc(description: "Cart ID")
+        quote_name: String @doc(description: "Quote name")
+        comment: String @doc(description: "Comment")
+        files: [FileInput] @doc(description: "Attached files")
```

Comment:
This should probably be `[FileInput!]`, right? Not sure of a use-case for a `null` in there

Reply:
Corrected this.